### PR TITLE
Added support for mysql on gcp

### PR DESCRIPTION
**Pull request by juandiegopalomino**
Screen Shot 2021-06-03 at 4 15 40 PM
Tested with default engine 5.7

### CODE REVIEW

File: config/tf_modules/gcp-mysql/main.tf [hunk 1]
```diff
@@ -0,0 +1,49 @@
+resource "random_id" "key_suffix" {
+  byte_length = 8
+}
+
+resource "random_password" "root_auth" {
+  length  = 20
+  special = false
+}
+
+# TODO: add encryption key name once out of beta
```
Comment (legalosLOTR) [inline config/tf_modules/gcp-mysql/main.tf:10]: what does this mean?

**Review by legalosLOTR — APPROVED** [review]
Can you also send out a pr for the relevant docs? :))

File: config/tf_modules/gcp-mysql/main.tf (hunk 1 above)
Comment (juandiegopalomino) [reply config/tf_modules/gcp-mysql/main.tf:10]: Disk encryption in GCP is currently in beta

File: config/tf_modules/gcp-mysql/outputs.tf [hunk 2]
```diff
@@ -0,0 +1,16 @@
+output "db_user" {
```
Comment (legalosLOTR) [inline config/tf_modules/gcp-mysql/outputs.tf:1]: Add a comment here that this list of outputs needs to be the same as the ones for gcp-postgres and should match the ones at `gcp_k8s_service.py:handle_rds_link()`

File: config/tf_modules/gcp-mysql/main.tf (hunk 1 above)
Comment (juandiegopalomino) [reply config/tf_modules/gcp-mysql/main.tf:10]: So I decided not to tangle with that and the _separate_ provider it would make us use

Comment (legalosLOTR) [reply config/tf_modules/gcp-mysql/main.tf:10]: Ah so is it currently encrypted or not?

Comment (legalosLOTR) [reply config/tf_modules/gcp-mysql/main.tf:10]: Please clarify in the comment that "disk encryption in gcp mysql is currently in beta"

File: config/tf_modules/gcp-mysql/outputs.tf (hunk 2 above)
Comment (juandiegopalomino) [reply config/tf_modules/gcp-mysql/outputs.tf:2]: done!

File: config/tf_modules/k8s-service/k8s-service/templates/deployment.yaml [hunk 3]
```diff
@@ -25,7 +25,7 @@ spec:
         tags.datadoghq.com/service: {{ include "k8s-service.serviceName" . }}-{{ include "k8s-service.namespaceName" . }}
         tags.datadoghq.com/version: {{ .Values.version | quote }}
       annotations:
-        config.linkerd.io/skip-outbound-ports: "8126,5432"
+        config.linkerd.io/skip-outbound-ports: "8126,5432,3306"
```
Comment (NitinAgg) [inline config/tf_modules/k8s-service/k8s-service/templates/deployment.yaml:28]: Let's add a comment for each of the ports, why we had to skip them

**Pull request merged**